### PR TITLE
French translation of user configuration page, mainly token paragraph.

### PR DESCRIPTION
**Pull request by aHenryJard**
I tried to check with @Wadeck PR https://github.com/jenkinsci/jenkins/pull/4985, I should not be on the same files.
Also tell me if I am wrong, but I think that special characters does not need to be escaped in html help files.

token translation

### Proposed changelog entries

* Entry 1: French translation for the token paragraph in user configuration.
* Entry 1: French translation of the root breadcrumb (Dashboard)

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change). [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  * Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@mention

### Maintainer checklist

Before the changes are marked as `ready-for-merge`: 

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are correct
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins-ci.org/issues/?filter=12146)).

### CODE REVIEW

File: core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+RevokeToken=R\u00e9voquer ce jeton
+CopyToken=Copier ce jeton
+NoTokenYet=Il n''y a pas de jeton enregistr\u00e9 pour cet utilisateur.
+TokenLastUse=Utilis\u00e9 <b>{0}</b> fois, le dernier usage date de <b>{1}</b> jours
+StatisticsDisabled=Pas de statistique disponible
+StatisticsDisabled.Title=Les statistiques d''utilisation du jeton sont d\u00e9sactiv\u00e9es
+TokenNeverUsed=Jamais utilis\u00e9
+TokenNeverUsedTooltip=Il n''y a pas de date de derni\u00e8re utilisation pour ce jeton
+TokenNeverUsed.Title=Il est fortement recommand\u00e9 de supprimer les jetons qui ne sont pas utilis\u00e9s
+ConfirmRevokeSingle=Confirmez-vous la suppression de ce jeton? Les applications l''utilisant ne pourront plus se \
```
Comment (Wadeck) [inline core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties:13]:
```suggestion
ConfirmRevokeSingle=Confirmez-vous la suppression de ce jeton ? Les applications l''utilisant ne pourront plus se \
```

Interrogation mark in french must have a space before contrary to the english ones.

File: core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  connecter.
+CurrentTokens=Jeton(s) actuel(s)
+TokenCreation=Cr\u00e9\u00e9 il y a {0} jours
+RenameToken=Enregistrer le nouveau nom de jeton
```
Comment (Wadeck) [inline core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties:17]: FYI this entry is never used

**Review by Wadeck — APPROVED** [review]
👍 Thanks @aHenryJard for the contribution

File: core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+TokenNeverUsed.Title=Il est fortement recommand\u00e9 de supprimer les jetons qui ne sont pas utilis\u00e9s
+ConfirmRevokeSingle=Confirmez-vous la suppression de ce jeton ? Les applications l''utilisant ne pourront plus se \
+  connecter.
+CurrentTokens=Jeton(s) actuel(s)
+TokenCreation=Cr\u00e9\u00e9 il y a {0} jours
+RenameToken=Enregistrer le nouveau nom de jeton
+LegacyToken=Nous recommandons de revoquer ce jeton obsol\u00e8te et de le remplacer par un nouveau jeton pour une \
+  meilleure s\u00e9curit\u00e9.
+NoLegacyToken=L''utilisateur n''as pas de jeton obsol\u00e8te
+RevokedToken=Le jeton obsolète a \u00e9t\u00e9 r\u00e9voqu\u00e9
```
Comment (Wadeck) [inline core/src/main/resources/jenkins/security/ApiTokenProperty/config_fr.properties:21]:
```suggestion
RevokedToken=Le jeton obsol\u00e8te a \u00e9t\u00e9 r\u00e9voqu\u00e9
```

**Comment by Wadeck** [comment]
> Also tell me if I am wrong, but I think that special characters does not need to be escaped in html help files.

Exactly :)

**Review by res0nance — APPROVED** [review]

File: core/src/main/resources/jenkins/security/Messages_fr.properties [hunk 4]
```diff
@@ -0,0 +1,3 @@
+# This file is under the MIT License by authors
+
+ApiTokenProperty.DisplayName=Jeton d''API (token)
```
Comment (alecharp) [inline core/src/main/resources/jenkins/security/Messages_fr.properties:3]:
```suggestion
ApiTokenProperty.DisplayName=Jeton d''API (token)

```

**Review by alecharp — APPROVED** [review]

**Comment by res0nance** [comment]
We may merge in 24h if there is no negative feedback

**Review by oleg-nenashev — APPROVED** [review]
Looks good! This is a Hacktoberfest pull request, right?

**Pull request merged**